frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -163,10 +180,19 @@ export const AccountsSummary = ({
     accounts.forEach(account => {
       const currentCode = account.code;
       subscriptions.push(statusChanged(account.code, () => currentCode === account.code && update(account.code)));
-      subscriptions.push(syncdone(account.code, () => currentCode === account.code && update(account.code)));
+      subscriptions.push(syncdone(account.code, () => {
+        if (currentCode === account.code) {
+          update(account.code);
+        }
+        getAccountSummary();
+        getAccountsBalance();
+        getAccountsTotalBalance();
+        getCoinsTotalBalance();
```

Comment:
Shouldn't it be moved inside the `update()` call, so it is also called when `statusChanged` fires? Not 100% sure but seems weird that we'd need to update the chart and account balances, but not the totals.

Reply:
Yeah I think that makes actually sense